{
	"next_comment_id": 4,
	"comment": [
		{
			"comment_id": 1,
			"body": "getting this error message on both zeromail and thunderwave:\n```\nInternal error: AttributeError: '_OpenSSL' object has no attribute 'digest_ecdsa_sha1'\nUiWebsocket.py line 85 > UiWebsocket.py line 234 > CryptMessage/CryptMessagePlugin.py line 30 > CryptMessage/CryptMessage.py line 10 > __init__.py line 17 > ecc.py line 20 > ecc.py line 323\n```",
			"post_id": 148,
			"date_added": 1565291840
		},
		{
			"comment_id": 2,
			"body": "btw is this error message important? since now logged errors are also displayed on zerohello and i didn't pay that much attention to errors on my log:\n\n```\n2019-10-16 07:12:21,613] ERROR - UiWSGIHandler error: ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine in UiServer.py line 59 > pywsgi.py line 924 > pywsgi.py line 910 > pywsgi.py line 757 > pywsgi.py line 778 > pywsgi.py line 740 > pywsgi.py line 704 > _socket3.py line 458 > _socketcommon.py line 374 > _socketcommon.py line 303 > _socket3.py line 439\n[2019-10-16 07:12:21,614] ERROR Ui.UiServer Error 500: UiWSGIHandler error: ConnectionAbortedError: None\n```",
			"post_id": 150,
			"date_added": 1571236933
		},
		{
			"comment_id": 3,
			"body": "> [nofish](#comment_331_1J3rJ8ecnwH2EPYa6MrgZttBNc61ACFiCj): Not really important, it means your browser did not closed the connection properly. What browser are you using? I will try to reproduce it and handle it as a not important error.\n\nfirefox 69.0.3 (latest)",
			"post_id": 150,
			"date_added": 1571254360
		}
	],
	"comment_vote": {},
	"post_vote": {
		"139": 1,
		"140": 1,
		"142": 1,
		"144": 1,
		"145": 1,
		"147": 1,
		"148": 1,
		"149": 1,
		"152": 1
	}
}